src/actions/SearchActions.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import request from 'superagent'
+import SearchConstants from '../constants/SearchConstants.js'
+
+export const requestCourses = (course: string) => ({
+  type: SearchConstants.REQUEST_COURSES,
+  query: course,
```

Comment:
nit: I remember reading that actions should use type and payload as keys for consistency – since payloads will have different models I guess this is fine but I wanted to double check that this is what we want to do for actions going forward.